src/CDSAPI.jl

```diff
@@ -7,7 +7,16 @@ using Base64
 export
     py2ju
 
-function retrieve(name, params, filename)
+"""
+    retrieve(name::AbstractString,
+        params::Dict,
```

Comment:
Please make sure you are not inserting real TAB in your text editor. Do you mind aligning the arguments and removing the type annotation?

Reply:
You mean I should configure the editor to insert 4 spaces instead of a TAB?

Reply:
Yes, TABs are discouraged in general.

Reply:
and should I remove type annotations from both docstrings and fuction defination?

Reply:
A bit late but done. I was in a session :grimacing: